external/vcm/vcm/cubedsphere/regridz.py

```diff
@@ -4,7 +4,15 @@
 
 from typing import Tuple, Union
 
-import vcm.mappm
```

Comment:
Action item here is that we're going to try to fix my environment and revisit it if necessary. Ideally not include this change (and the one for xgcm).

Reply:
Fixed, reverting these changes.